SnapSpire/Features/Discover/View/DiscoverView.swift

```diff
@@ -0,0 +1,35 @@
+import SwiftUI
+
+struct DiscoverView: View {
+    var body: some View {
+        NavigationView {
```

Comment:
What is the purpose of the NavigationView here? I don't see any navigation link / destination

Reply:
I added the navigation link in SectionListView